pkg/operator/controller/garden/reconciler_reconcile.go

```diff
@@ -224,6 +225,22 @@ func (r *Reconciler) deployEtcdsFunc(garden *operatorv1alpha1.Garden, etcdMain,
 			}
 		}
 
+		// Deploy NetworkPolicy allowing ETCD to talk to the runtime cluster's API server.
```

Comment:
I didn't understand why there's any communication between virtual garden ETCD and runtime cluster's API server. Don't all the queries to this ETCD go through the virtual garden API server?

Reply:
ETCD and backup-restore communicate with the runtime cluster API server to maintain e.g. the `Lease` objects (maybe more). Also, backup-restore must be able to talk to the world anyways to reach the backup store.